spec/basic.spec.js

```diff
@@ -91,6 +91,43 @@ describe('Schemy basic validations', function() {
 		expect(result).toBe(true);
 	});
 
+	it("Should pass validation with correct child schemas using strict property", function () {
+    const nameSchema = new Schemy(
```

Comment:
Thank you for your PR! In order to merge it into the codebase, can you please fix indentation here? Schemy uses tab with 4 width. There is a .editorconfig to make this easier to solve in the root folder

Reply:
Thanks for the review! do you know when will this new version be available??